modules/packer/custom-image/image.pkr.hcl

```diff
@@ -70,6 +70,8 @@ locals {
     : local.on_host_maintenance_default
   )
 
+  accelerator_type = var.accelerator_type == null ? null : "projects/${var.project_id}/zones/${var.zone}/acceleratorTypes/${var.accelerator_type}"
```

Comment:
Is this change related to the "packaging" ? Reply to this comment is enough, no need to update PR.

Reply:
No, it's a separate commit I'm adding because it's related to Packer generally.